src/lib/sqle/query/query.service.ts

```diff
@@ -0,0 +1,313 @@
+/*
+ * Copyright (C) 2016-2017 by Teradata Corporation. All rights reserved.
+ * TERADATA CORPORATION CONFIDENTIAL AND TRADE SECRET
+ */
+
+import { Injectable, Provider, Optional, SkipSelf } from '@angular/core';
+import { HttpClient, HttpHeaders, HttpErrorResponse } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { map, catchError } from 'rxjs/operators';
+
+/*
+ * These interfaces are duplicated within the query service and the system service.
+ * TODO: DRY this up
+ */
+enum SystemType {
+  Teradata = 'TERADATA',
+  Aster = 'ASTER',
+  Presto = 'PRESTO',
```

Comment:
Do we support presto anymore? And do these services in general pertain to Teradata?

Reply:
Yeah, we dont support Presto anymore. For the purpose of the `SQLE` package.. lets remove Aster and Presto from the copied interface.